app/views/shared/_contextual_result_info.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="alert alert-info alert-dismissible fade show" role="alert">
+<div class="result_info_message alert alert-info alert-dismissible fade show" role="alert">
```

Comment:
Is the `result_info_message` class used, or is it just there for our benefit? 

```suggestion
<div class="result-info-message alert alert-info alert-dismissible fade show" role="alert">
```

I think we prefer dashes in class names?

Reply:
I added the class here for parity between the two alerts and I wanted the class on the date sort alert to select for testing 🤷‍♂️ . I'll switch to dashes